spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -208,7 +208,7 @@ The target Spring Cloud Stream application starter should be re-packaged to supp
 The JDBC-sink can be used to insert message payload data into a relational database table. By default,
 it inserts the entire payload into a table named after `spring.application.name` value set at the
 application level, and if it is not set, by default the application expects the table with the name
-`messages`. To alter this behavior, the JDBC sink accepts link:https://github.com/spring-cloud-stream-app-starters/jdbc/blob/master/spring-cloud-starter-stream-sink-jdbc/README.adoc[several options] that you can pass using the --foo=bar notation in the stream, or change globally. You should also change the `jdbc.initialize` property to true to have this script execute when the sink starts up.
```

Comment:
should we link to docs rather than README? Maybe use -- http://docs.spring.io/spring-cloud-stream-app-starters/docs/current/reference/html/spring-cloud-stream-modules-sinks.html#spring-cloud-stream-modules-jdbc-sink